### PR TITLE
add DarshGupta1910

### LINKED ISSUE #742: DarshGupta1910
screenbud-d77cdb4f-c68a-4488-a614-e8cd33a7561d (1)

### PR DESCRIPTION
**Pull request by DarshGupta1910**
Resolve #742

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1378,5 +1378,11 @@
     "githubUsername": "Vanshikapandey30",
     "imageUrl": "https://user-images.githubusercontent.com/70807500/135111984-f6c4f540-949b-451c-8fb3-22d5daa07575.png",
     "issueNumber": 738
+  },
+    {
```
Comment (CodingSpecies) [inline data.json:1382]: Please can you tab this back a space, so it is in line with the other brackets?

**Review by CodingSpecies — CHANGES_REQUESTED** [review]
@DarshGupta1910 please see the suggestion:

File: data.json (hunk 1 above)
Comment (DarshGupta1910) [reply data.json]: done

**Review by CodingSpecies — APPROVED** [review]
LGTM, thank you for adding the changes @DarshGupta1910

File: data.json (hunk 1 above)
Comment (CodingSpecies) [reply data.json]: Thank you  @DarshGupta1910

**Review by schmelto — APPROVED** [review]

**Pull request merged**